test/helpers/abi.ts

```diff
@@ -0,0 +1,10 @@
+import fs from "fs";
+import chai from "chai";
+const { expect } = chai;
+
+export const parseABI = (abiPath) => {
+  expect(fs.existsSync(abiPath)).to.eq(true);
```

Comment:
since this is a helper method, I think throwing an error would be more clear than a false expectation